src/components/PetForm.js

```diff
@@ -23,7 +23,7 @@ export default class PetForm extends Component {
       sex: 'Female',
       age: '',
       size: '',
-      picture: '',
+      profileImg: '',
```

Comment:
What was the reason behind changing this variable name? Does it need to be changed elsewhere as well?

Reply:
I changed it because the state we use for the file name is now 'profileImg' and I prefer to have a default value (though not necessary).

Reply:
Makes sense, thanks for making the change!